tools/e2e/setup.mjs

```diff
@@ -1,22 +1,15 @@
-import { appendFileSync, existsSync, promises, readFileSync, writeFileSync } from 'fs'
+import { appendFile, mkdtemp, readFile, rm, writeFile } from 'fs/promises'
```

Comment:
I updated this file to use top-level await and async version of the fs functions.